.github/workflows/test.yml

```diff
@@ -6,6 +6,9 @@ on:
       - main
   pull_request:
 
+env:
+  SKIP_CI: TRUE
```

Comment:
Is this still used? I see it used later in the CI yaml, but not in the Go code.

Reply:
Great question! it is used for the `Makefile`. This can be convenient as if we want to add more tags to the test run in the future but not change the .yml as the downstream dep along the way

https://github.com/celestiaorg/celestia-node/pull/578/files#diff-faff1af3d8ff408964a57b2e475f69a6b7c7b71c9978cccc8f471798caac2c88R45